osu.Game/Input/Bindings/DatabasedKeyBindingInputManager.cs

```diff
@@ -18,7 +19,9 @@ public abstract class DatabasedKeyBindingInputManager<T> : KeyBindingInputManage
 
         private readonly int? variant;
 
-        private BindingStore store;
+        private KeyBindingStore store;
+
+        public override IEnumerable<KeyBinding> DefaultMappings => ruleset.CreateInstance().GetDefaultKeyBindings();

```

Comment:
In one place it's called "keybindings" and in one "mappings". That seems strange.